### DISCUSSION TITLE
Shiny Dashboard warning : Text to be written must be a length-one character vector

### DISCUSSION THREAD

**Discussion by DiekiRianMustapa** [teamalgoritma/community#290] [Q&A] [answered]
Bisa membantu kah saya sedang memuat dashboard menggunakan shiny dengan source data dari google spread sheet 
secara rmd sudah berjalan dengan normal untuk codingnya namun saat dibuat shiny dashboard selalu muncul error seperti ini 

![image](https://user-images.githubusercontent.com/107328516/188797318-9f75a2cd-638d-448f-a268-c465619252a8.png)

Berikut link file shiny dashboard nya: 
https://github.com/DiekiRianMustapa/PCT

**Comment by onlyphantom** [comment, marked as answer]
1. Can you paste the output of `packageVersion("shiny)`? Shiny 1.7+ uses the latest font awesome icons v5.15. In this version, some old names are deprecated. You can find the names for font-awesome icons v5 here: https://fontawesome.com/v5/icons/

2. Try to set  `verify_fa=FALSE` too to isolate the problem a bit more, for example: `icon("fa-github-square", verify_fa = FALSE)` for all your icons. See if the problem persist?

**Reply by DiekiRianMustapa** [reply]
i use 1.7.2 for shiny package

**Comment by NabiilahArdini** [comment]
Halo @DiekiRianMustapa, untuk menjaga agar forum diskusi tetap berjalan kondusif, kami izin melakukan close pada diskusi ini karena belum ada respon setelah 7 hari respon terakhir. Semoga dapat dipahami.
